src/utils/rocksdb_serialization.hpp

```diff
@@ -30,6 +31,47 @@ namespace memgraph::utils {
 static constexpr const char *outEdgeDirection = "0";
 static constexpr const char *inEdgeDirection = "1";
 
+namespace {
+struct StartEndPositions {
+  size_t start;
+  size_t end;
+
+  size_t Size() const { return end - start; }
+  bool Valid() const { return start != std::string::npos && start <= end; }
+};
+
+template <typename T>
+inline std::string_view FindPartOfStringView(const std::string_view str, const char delim, T partNumber) {
+  StartEndPositions startEndPos{0, 0};
+  for (int i = 0; i < partNumber; ++i) {
+    startEndPos.start = startEndPos.end;
+    startEndPos.end = str.find(delim, startEndPos.start);
+    if (i < partNumber - 1) {
+      if (startEndPos.end == std::string::npos) {
+        // We didn't find enough parts.
+        startEndPos.start = std::string::npos;
+        break;
+      }
+      ++startEndPos.end;
+    }
+  }
+  return startEndPos.Valid() ? str.substr(startEndPos.start, startEndPos.Size()) : str;
+}
+
+inline std::string_view GetViewOfFirstPartOfSplit(const std::string_view src, const char delimiter) {
+  return FindPartOfStringView(src, delimiter, 1);
+}
+
+inline std::string_view GetViewOfSecondPartOfSplit(const std::string_view src, const char delimiter) {
+  return FindPartOfStringView(src, delimiter, 2);
+}
+
+inline std::string_view GetViewOfThirdPartOfSplit(const std::string_view src, const char delimiter) {
+  return FindPartOfStringView(src, delimiter, 3);
+}
+
+}  // namespace
+
 /// TODO: try to move this to hpp files so that we can follow jump on readings
```

Comment:
Remove what?

Reply:
I think this is one of your comments. Currently, it's unclear to me. Maybe we should update the TODO or remove it?